packages/jaeger-ui/src/components/QualityMetrics/CountCard.test.js

```diff
@@ -13,24 +13,34 @@
 // limitations under the License.
 
 import React from 'react';
-import { shallow } from 'enzyme';
-
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
 import CountCard from './CountCard';
 
 describe('CountCard', () => {
   const count = 108;
   const title = 'Test Title';
+  const examples = ['Examples'];
 
   it('renders null when props.count or props.title is absent', () => {
-    expect(shallow(<CountCard count={count} />).type()).toBe(null);
-    expect(shallow(<CountCard title={title} />).type()).toBe(null);
+    const { container: containerWithoutCount } = render(<CountCard count={count} />);
```

Comment:
I don't follow - why is this "WithoutCount" if you're passing `count={count}`?

Reply:
done